README.md

```diff
@@ -168,26 +168,45 @@ cp solr_configs/conf/schema.xml solr_scripts/schema.xml
 
 The Dockerized versions of Solr on the server need your attention, too. Follow these steps.
 
-1. Deploy the schema changes on the staging server. 
-2. Shell into the server, go to the `lametro-staging` repo, and remove and rebuild the Solr container.
+1. Deploy the schema changes on the staging server.
+2. Shell into the server as the ubuntu user, go to the `lametro-staging` repo, and remove and rebuild the Solr container.
 
 ```
-docker rm lametro-staging-solr
-docker-compose up solr-staging
+cd lametro-staging
+
+# we need to stop the container before removing it!
+sudo docker stop lametro-staging-solr  
+sudo docker rm lametro-staging-solr
+
+# -d is the daemon flag
+sudo docker-compose up -d solr-staging
+```
+
+3. Logout as the ubuntu user, then log in as the datamade user.
```

Comment:
Using `ssh ubuntu@ domain.name` to shell into the server starts your session as the Ubuntu user. You can reword this step to: 

"Switch to the datamade user: `sudo su - datamade`